nixos/modules/services/monitoring/prometheus/sachet.nix

```
@@ -0,0 +1,92 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.sachet;
+  configFile = pkgs.writeText "sachet.yml" (builtins.toJSON cfg.configuration);
+
+  # sachet does not allow parsing of env variables in its configuration file
+  runScript = pkgs.writeShellScript "run.sh" ''
+    ${pkgs.envsubst}/bin/envsubst -i "${configFile}" > /tmp/sachet.yaml
+    ${pkgs.prometheus-sachet}/bin/sachet -config /tmp/sachet.yaml -listen-address $1
+  '';
+in
+{
+  options = {
+    services.prometheus.sachet = {
+      enable = mkEnableOption "Sachet, an SMS alerting tool for the Prometheus Alertmanager";
+
+      configuration = mkOption {
+        type = types.nullOr types.attrs;
+        default = null;
+        example = literalExample ''
+          {
+            providers = {
+              twilio = {
+                # environment variables gets expanded at runtime
+                account_sid = "$TWILIO_ACCOUNT";
+                auth_token = "$TWILIO_TOKEN";
+              };
+            };
+            templates = [ ./some-template.tmpl ];
+            receivers = [{
+              name = "pager";
+              provider = "twilio";
+              to = [ "+33123456789" ];
+              text = "{{ template \"message\" . }}";
+            }];
+          }
+        '';
+        description = ''
+          Sachet configuration's as a nix attribute set.
```

Stole this from Postgres, maybe it sounds better?

```suggestion
          Sachet's configuration, specified as an attribute set.
```